SConscript.firmware

```diff
@@ -424,7 +425,9 @@ program_bin = env.Command(
     target='firmware.bin',
     source=program_elf,
     action=[
-        '$OBJCOPY -O binary -j .vendorheader -j .header -j .flash -j .data $SOURCE $TARGET',
+        '$OBJCOPY -O binary -j .vendorheader -j .header -j .flash -j .data --pad-to 0x08100000 $SOURCE ${TARGET}.p1',
```

Comment:
I don't like this `0x08100000` constant being here, but this was the most straightforward way how to do it.

Maybe there are some options to OBJCOPY which obsolete that and maybe also we can drop the CAT usage too.

Reply:
I think it's OK for now.